app/templates/search/documents.hbs

```diff
@@ -0,0 +1,98 @@
+{{#in-element this.customFiltersElement insertBefore=null}}
+  {{#if (user-may "search-confidential-documents")}}
+    <Search::ConfidentialOnly
+      @objectVerb={{t "documents"}}
+      @confidentialOnly={{this.confidentialOnly}}
+      @onChange={{set this "confidentialOnly"}}
+    />
+  {{/if}}
+  <Search::DocumentTypeFilter
+    @selected={{this.documentTypesBuffer}}
+    @onChange={{this.setDocumentTypes}}
+  />
+{{/in-element}}
+<div class="au-o-box au-o-box--small">
+  {{#if this.model.length}}
+    <Search::ResultsHeader
+      @pageSize={{this.size}}
+      @pageIdx={{this.page}}
+      @nbOfItems={{this.model.length}}
+      @searchText={{this.searchText}}
+      @searchType={{t "cases"}}
+      @onSortChange={{this.selectSort}}
+      @sortOptions={{this.sortOptions}}
+      @sort={{this.sort}}
+    />
+    <KDataTable
+      data-test-route-search-cases-data-table={{true}}
```

Comment:
copy paste test selector, can be removed.
We will add them when tests are being written.